swap_meet/clothing.py

```diff
@@ -1,2 +1,17 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0, age = 0):
+        super().__init__()
+        self.category = "Clothing"
+        self.condition = condition if condition else condition == 0
+        self.age = age if age else age == 0
```

Comment:
While the parent constructor is initialized, the benefits of inheritance are not being utilized here. The parent constructor allows the child class to add values to the inherited attributes from the parent. Instead what's happening here is that `Clothing` is re-creating category/condition attributes and not inheriting category/condition from the parent. We can reduce this entire constructor code to 2 lines by using inheritance:

```suggestion
class Clothing(Item):
    def __init__(self, condition = 0, age = 0):
        super().__init__(category="Clothing, condition = condition, age = age)
```